Zeeguu Reader/Zeeguu Reader/UpdateTranslationViewController.swift

```diff
@@ -59,6 +59,17 @@ class UpdateTranslationViewController: UITableViewController, UIPopoverPresentat
 		if let dict = action.getActionInformation() {
 			if let ot = dict["otherTranslations"] where !ot.isEmpty {
 				self.otherTranslations = JSON.parse(ot).dictionaryObject as? [String: String]
+				
+				if let ot = otherTranslations where ot.count > 0 {
+					s1.removeAll()
+					for (_, value) in ot {
+						s1.append(value)
+					}
+				}
+				
+				s1.sortInPlace({ $0.0.lowercaseString < $0.1.lowercaseString })
```

Comment:
I'll provide to you an "likelihood" property for each translation, so you can sort based on that,